### DISCUSSION TITLE
YetiForce Register

### DISCUSSION THREAD

**Discussion by DeMezzo** [YetiForceCompany/YetiForceCRM#15602] [General]
Hello, I'm new here and I've installed YetiForce. I actually like it a lot. There are only a few things that bothers me.

Whenever I log in, this registration comes up that the system is not registered, then I go to send and the message keeps coming back. Does it even work?

The next thing is that after logging in, you are always asked for what reason you are logging in. You always have to enter something and click on Save. Can't you turn it off, that's really annoying. You want to work with this system and not play. So this question is very pointless.

Then one more thing bothers me and that is that there is no APP for Android yet, is that planned or will there never be one?

It would be better if there would also be an APP to view the data offline. If you have no internet reception with your smartphone, but you can see the data and make new entries and as soon as the smartphone is online again, it updates itself.

It would be great if you can help me with my problem as soon as possible.

greeting

Mezzo

**Comment by mariuszkrzaczkowski** [comment]
> Whenever I log in, this registration comes up that the system is not registered, then I go to send and the message keeps coming back. Does it even work?

Yes, it works. What version of the system do you have? Attach a dump of the server configuration.

> The next thing is that after logging in, you are always asked for what reason you are logging in. You always have to enter something and click on Save. Can't you turn it off, that's really annoying. You want to work with this system and not play. So this question is very pointless.

See #13883

> Then one more thing bothers me and that is that there is no APP for Android yet, is that planned or will there never be one?

Currently, we do not plan an Android application, the system is fully responsive so you can easily use it on your phone.
Although we don't say completely no, if someone is interested, we can create.

> It would be better if there would also be an APP to view the data offline. If you have no internet reception with your smartphone, but you can see the data and make new entries and as soon as the smartphone is online again, it updates itself.

We know that, we have a lot of ideas like this, but no one wants to pay to do it.

> It would be great if you can help me with my problem as soon as possible.

All your questions are answered in issues or discussions, these threads have already been discussed

**Comment by DeMezzo** [comment]
I have version 6.3.
How can I complete the registration today?
What is the registration supposed to be good for anyway?
greeting
Mezzo

**Reply by mariuszkrzaczkowski** [reply]
> How can I complete the registration today?

send APP ID to e-mail address registration@yetiforce.com

> What is the registration supposed to be good for anyway?

This is the system license, https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/licenses/LicenseEN.txt

**Comment by DeMezzo** [comment]
Okay e-mail has been sent

**Reply by mariuszkrzaczkowski** [reply]
Accepted check status

**Comment by DeMezzo** [comment]
Perfect thank you.

I am now starting to learn the system.
________________________________
Von: Mariusz Krzaczkowski ***@***.***>
Gesendet: Montag, 3. Januar 2022 13:34
An: YetiForceCompany/YetiForceCRM ***@***.***>
Cc: DeMezzo ***@***.***>; Author ***@***.***>
Betreff: Re: [YetiForceCompany/YetiForceCRM] YetiForce Register (Discussion #15602)


Accepted check status

—
Reply to this email directly, view it on GitHub<https://github.com/YetiForceCompany/YetiForceCRM/discussions/15602#discussioncomment-1899721>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXEBLH4K5YFA76HQM7EFVPTUUGJT3ANCNFSM5LECSGWA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
